python-sdk/nuscenes/eval/detection/README.md

```diff
@@ -122,51 +130,51 @@ Our final score is a weighted sum of mean Average Precision (mAP) and several Tr
-This is particularly important as some TP metrics may decrease with increasing recall values. 
+The metric is averaged over the same recall thresholds as for mAP. 
+If a recall value > 0.1 is not achieved, the TP error for that class is set to 1.
+
 Finally we compute the mean over classes.
+
 * **mean Average Translation Error (mATE)**: For each match we compute the translation error as the Euclidean center distance in 2D in meters.
 * **mean Average Scale Error (mASE)**: For each match we compute the 3D IOU after aligning orientation and translation.
 * **mean Average Orientation Error (mAOE)**: For each match we compute the orientation error as the smallest yaw angle difference between prediction and ground-truth in radians.
 * **mean Average Velocity Error (mAVE)**: For each match we compute the absolute velocity error as the L2 norm of the velocity differences in 2D in m/s.
 * **mean Average Attribute Error (mAAE)**: For each match we compute the attribute error as as *1 - acc*, where acc is the attribute classification accuracy of all the relevant attributes of the ground-truth class. The attribute error is ignored for annotations without attributes.
 
+All errors are >= 0, but note that for translation and velocity errors the errors are unbounded, and can be any positive value.
+
+
 ### Weighted sum metric
 * **Weighted sum**: We compute the weighted sum of the above metrics: mAP, mATE, mASE, mAOE, mAVE and mAAE.
-For each error metric x (excl. mAP), we use *1 - x*.
-We assign a weight of *5* to mAP and *1* to the 5 TP metrics.
-Then we normalize by 10.
+As a first step we convert the TP errors to TP scores as *x_score = max(1 - x_err, 0.0)*.
+We then assign a weight of *5* to mAP and *1* to the 5 TP scores, sum and normalize by 10.
+
 
 ## Leaderboard & challenge tracks
 Compared to other datasets and challenges, nuScenes will have a single leaderboard for the detection task.
 For each submission the leaderboard will list method aspects and evaluation metrics.
 Method aspects include input modalities (lidar, radar, vision), use of map data and use of external data.
 To enable a fair comparison between methods, the user will be able to filter the methods by method aspects.
-The user can also filter the metrics that should be taken into account for the weighted sum metric. 
-
-We define two such filters here.
+ 
+We define three such filters here.
 These filters correspond to the tracks in the nuScenes detection challenge.
 Methods will be compared within these tracks and the winners will be decided for each track separately:
 
 * **LIDAR detection track**: 
 This track allows only lidar sensor data as input.
-It is supposed to be easy to setup and support legacy lidar methods.
 No external data or map data is allowed.
 
-* **Open detection track**: 
+* **VISION detection track**: 
+This track allows only camera sensor data (images) as input.
+No external data or map data is allowed.
```

Comment:
This excludes pretraining a Neural Network for image classification (same for lidar detection track), but not object detection or similar tasks with annotated object locations.